data.sql

```diff
@@ -120,4 +120,86 @@ UPDATE animals
 
+---------------------------------------------------------------------------------------------
+
+-- Insert the following data for vets:
+-- Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
+-- Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
+-- Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
+-- Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23'),
+       ('Maisy Smith', 26, '2019-01-17'),
+       ('Stephanie Mendez', 64, '1981-05-04'),
+       ('Jack Harkness', 38, '2008-06-08');
+
+---------------------------------------------------------------------------------------------
+
+-- Insert the following data for specialties:
+-- Vet William Tatcher is specialized in Pokemon.
+-- Vet Stephanie Mendez is specialized in Digimon and Pokemon.
+-- Vet Jack Harkness is specialized in Digimon.
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES (1, 2),
+       (3, 1),
+       (3, 2),
+       (4, 1);
+
+---------------------------------------------------------------------------------------------
 
+-- Insert the following data for visits:
+-- Agumon visited William Tatcher on May 24th, 2020.
+-- Agumon visited Stephanie Mendez on Jul 22th, 2020.
+-- Gabumon visited Jack Harkness on Feb 2nd, 2021.
+-- Pikachu visited Maisy Smith on Jan 5th, 2020.
+-- Pikachu visited Maisy Smith on Mar 8th, 2020.
+-- Pikachu visited Maisy Smith on May 14th, 2020.
+-- Devimon visited Stephanie Mendez on May 4th, 2021.
+-- Charmander visited Jack Harkness on Feb 24th, 2021.
+-- Plantmon visited Maisy Smith on Dec 21st, 2019.
+-- Plantmon visited William Tatcher on Aug 10th, 2020.
+-- Plantmon visited Maisy Smith on Apr 7th, 2021.
+-- Squirtle visited Stephanie Mendez on Sep 29th, 2019.
+-- Angemon visited Jack Harkness on Oct 3rd, 2020.
+-- Angemon visited Jack Harkness on Nov 4th, 2020.
+-- Boarmon visited Maisy Smith on Jan 24th, 2019.
+-- Boarmon visited Maisy Smith on May 15th, 2019.
+-- Boarmon visited Maisy Smith on Feb 27th, 2020.
+-- Boarmon visited Maisy Smith on Aug 3rd, 2020.
+-- Blossom visited Stephanie Mendez on May 24th, 2020.
+-- Blossom visited William Tatcher on Jan 11th, 2021.
+
+INSERT INTO visits (vet_id, animal_id, visit_date)
+VALUES
+  -- Agumon
+  (1, 1, '2020-05-24'),
+  (3, 1, '2020-07-22'),
+  -- Gabumon
+  (4, 2, '2021-02-02'),
+  -- Pikachu
+  (2, 3, '2020-01-05'),
+  (2, 3, '2020-03-08'),
+  (2, 3, '2020-05-14'),
+  -- Devimon
+  (3, 4, '2021-05-04'),
+  -- Charmander
+  (4, 14, '2021-02-24'),
+  -- Plantmon
+  (1, 6, '2019-12-21'),
+  (1, 6, '2020-08-10'),
+  (2, 6, '2021-04-07'),
+  -- Squirtle
+  (3, 7, '2019-09-29'),
+  -- Angemon
+  (4, 8, '2020-10-03'),
+  (4, 8, '2020-11-04'),
+  -- Boarmon
+  (2, 9, '2019-01-24'),
+  (2, 9, '2019-05-15'),
+  (2, 9, '2020-02-27'),
+  (2, 9, '2020-08-03'),
+  -- Blossom
+  (1, 10, '2020-05-24'),
+  (1, 10, '2021-01-11');
```

Comment:
- [ ] Also, you can dynamically add the various association instead of manually. Like so 👇🏾 


```sql
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Gabumon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-02');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-01-05');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-03-08');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-05-14');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Devimon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2021-05-04');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Charmander'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-24');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-12-21');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-08-10');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2021-04-07');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Squirtle'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2019-09-29');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-10-03');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-11-04');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-01-24');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-05-15');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-02-27');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-08-03');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-05-24');
 INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
```